kernel/kernel-api/src/test/scala/io/delta/kernel/internal/SnapshotManagerSuite.scala

```diff
@@ -836,28 +805,17 @@ class SnapshotManagerSuite extends AnyFunSuite with MockFileSystemClientUtils {
   test("getLogSegmentForVersion: corrupt _last_checkpoint with empty delta log") {
     val exMsg = intercept[InvalidTableException] {
       snapshotManager.getLogSegmentForVersion(
-        createMockFSListFromEngine(Seq.empty),
-        Optional.of(1),
+        mockEngine(
+          jsonHandler = new MockReadLastCheckpointFileJsonHandler(
+            s"$logPath/_last_checkpoint", 1),
+          fileSystemClient = new MockListFromFileSystemClient(_ => Seq.empty)
+        ),
```

Comment:
Sorry why can't this use `createMockFSAndJsonEngineForLastCheckpoint`?

Reply:
Great catch! My sunday brain missed this!